### ISSUE TITLE
Support text wrap for titles and subtitles

### ISSUE THREAD

**Issue by ndrezn** [plotly/plotly.py#4819]
Titles and subtitles in Plotly charts overflow by default. 

With subtitle support, having long strings in this area is more common (and more useful), so we should automatically wrap text if it gets too long in the title or subtitle area.

![Image](https://github.com/user-attachments/assets/42abf81d-2549-401e-8774-359924cf3009)

**Comment by ndrezn** [comment]
Yeah, `<br>` is a solid workaround, but in the context of responsive design we want automatic text wrapping.

**Comment by eliasdabbas** [comment]
Oh, I didn't think of the effects on responsiveness. Great point.
I realize it can be tough to safely add more HTML tags :) 
Thanks!

**Comment by kburchfiel** [comment]
Yes, this would be a huge help! For instance, it's great that maps and map legends scale automatically to fit whatever display I'm using (provided that I don't fix the width and height). However, when creating maps that will be seen both on mobile devices and desktops, I don't have a great option for longer map titles: if I add in `<br>` tags to wrap them, they may look strange on desktop browsers--but if I exclude the `<br>` tags, they'll overflow mobile displays.

**Comment by kburchfiel** [comment]
This would also be a great help for Plotly.js. However, with JavaScript code, I can at least create a title as a separate HTML element (which wraps automatically to fit the window's size, in my experience).
